api/src/main/java/marquez/db/migrations/V56_1__FacetViews.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2018-2022 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package marquez.db.migrations;
+
+import lombok.extern.slf4j.Slf4j;
+import org.flywaydb.core.api.MigrationVersion;
+import org.flywaydb.core.api.migration.Context;
+import org.flywaydb.core.api.migration.JavaMigration;
+import org.jdbi.v3.core.Jdbi;
+
+/**
+ * Dataset migration written in Java because SQLs used here will be reused in next migration when
+ * processing content of `lineage_events` table to facets' tables.
+ */
+@Slf4j
+public class V56_1__FacetViews implements JavaMigration {
+
+  public static final String DATASET_FACETS_DEFINITION =
```

Comment:
Migration is written as Java code because facet definitions will be reused in a migration that writes data to facets table.